### PR TITLE
Fix test_qpt_ml() and run it.

### PR DESCRIPTION
**Pull request by blakejohnson**
Note that this test still needs some calls to `@test` to verify more than "it
runs".

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -87,25 +87,25 @@ end
...
+    prep = map(projector,[ [1,0], [0,1], 1/sqrt(2)*[1,1], 1/sqrt(2)*[1,-1], 1/sqrt(2)*[1,-1im], 1/sqrt(2)*[1,1im] ] )
+
+    exps = [ choi_liou_involution(vec(o)*vec(p)') for o in obs, p in prep ]
 
     pred = reduce(vcat, map(m->vec(m)', vec(exps)) )
-    
+
     if trace(ρ) == 0.0
-        ρ = choi_liou_involution(liou(rand_unitary(2))) 
+        ρ = choi_liou_involution(liou(rand(RandomQuantum.ClosedHaarEnsemble(2))))
```
Comment (blakejohnson) [inline test/runtests.jl]: @marcusps Did I choose the correct distribution?

Comment (marcusps) [inline test/runtests.jl]: If you wanted a Haar random unitary acting on a 2D space, yes.

**Comment by marcusps** [comment]
The (dead) QPT code is a few generations older than the QST code, and needs more extensive revamping. 

That being said, I suppose this is a first step, but I am not sure why it is failing on Travis, since it does not fail for me locally.

File: test/runtests.jl (hunk 1 above)
Comment (blakejohnson) [inline test/runtests.jl]: Well, I have no idea was `rand_unitary` did previously.

**Comment by blakejohnson** [comment]
I was seeing various failures locally when I tried to run the tests until I manually calling `using X...` on all the test dependencies. It seems to be related to pre-compilation in some way.

**Comment by blakejohnson** [comment]
So, I *think* what is going on is that SchattenNorms.jl declares `__precompile__()` but Convex.jl does not, and this is not allowed.

**Comment by marcusps** [comment]
Strangely, `SchattenNorms.jl` builds without a hitch (at least on 0.5 and the nightly)

**Comment by marcusps** [comment]
This pretty old and neglected code (the QPT portion anyway) so I have to
say I'm not sure what it did anymore either. Random Haar unitary should do
it.

**Comment by blakejohnson** [comment]
Ok, finally passing tests.

**Comment by marcusps** [comment]
Have a look at #2

**Comment by blakejohnson** [comment]
Closing in favor of #2.

**Pull request closed without merging**